include/magic_enum.hpp

```diff
@@ -112,11 +112,31 @@ MAGIC_ENUM_USING_ALIAS_STRING_VIEW
 using std::string_view;
 #endif
 
+using char_type = string_view::value_type;
+
 // If need another string type, define the macro MAGIC_ENUM_USING_ALIAS_STRING.
 #if defined(MAGIC_ENUM_USING_ALIAS_STRING)
 MAGIC_ENUM_USING_ALIAS_STRING
 #else
-using std::string;
+using string = std::basic_string<char_type>;
```

Comment:
see https://github.com/Neargye/magic_enum/issues/80

Reply:
Thanks! I solve it with reverting this modification.

Example usage in `test_wchar_t.cpp`